eng/scripts/TypeSpec-Validation.ps1

```diff
@@ -15,7 +17,24 @@ if ($typespecFolders) {
   $typespecFolders = $typespecFolders.Split('',[System.StringSplitOptions]::RemoveEmptyEntries)
   foreach ($typespecFolder in $typespecFolders) {
     LogGroupStart "Validating $typespecFolder"
+
+    if ($CheckAll) {
+      $suppression = Get-Suppression "TypeSpecValidationAll" $typespecFolder
+      if ($suppression) {
+        $reason = $suppression["reason"] ?? "<no reason specified>"
```

Comment:
Do you think it is worth putting this default into the core suppression tool?

Reply:
I see what you mean because it's a little bit of duplicate code.  However, I think it's a tricky question, how much the core `get-suppressions` tool (both the CLI and the JS function) should respect what's actually in `suppressions.yaml`, versus adding defaults like a `reason` text if none is specified.

For now, I think leave it to clients, what to do if the `reason` is unspecified.  If we wanted to make any change in this area, we could consider requiring `reason` in the `suppressions.yaml`, so spec authors **always** need to set some reason string.  It's currently optional (but strongly recommended).